### PR TITLE
look at form instead of opts

### PR DESCRIPTION
**Pull request by emord**
@orangejenny will fix your `rtl` issue

### CODE REVIEW

File: src/javaRosa/util.js [hunk 1]
```diff
@@ -252,7 +252,8 @@ define([
     var insertOutputRef = function(vellum, target, path, mug, dateFormat) {
         var output = getOutputRef(path, dateFormat),
             form = vellum.data.core.form;
-        if ((!mug && vellum.opts().features.rich_text) || (mug && mug.supportsRichText())) {
+        if ((!mug && form.useRichText !== false) ||
```
Comment (orangejenny) [inline src/javaRosa/util.js]: Is it possible to initialize `form.useRichText` somewhere so we could get away from doing strict equality checks on booleans?

Comment (millerdev) [inline src/javaRosa/util.js]: yes please

Comment (millerdev) [inline src/javaRosa/util.js]: I thought someone had added that to the [form constructor](https://github.com/dimagi/Vellum/blob/bad7a8faa5b58cb27fcf5c2c2230d32c80378481/src/form.js#L111) a while ago. That seems like an ideal place for it.

**Comment by millerdev** [comment]
gtm. not sure if you want to address @orangejenny's comment here or in a different PR

File: src/javaRosa/util.js (hunk 1 above)
Comment (orangejenny) [inline src/javaRosa/util.js]: Sorry, original comment wasn't really relevant - forgot that https://github.com/dimagi/Vellum/pull/667/files#diff-62acef2590804164cb77b4b5aced00ebR141 initializes it but hasn't been merged yet.

**Pull request merged**